src/storybook/pages/home/home-lean-logged-in.ts

```diff
@@ -0,0 +1,262 @@
+import type { Args } from '@storybook/web-components';
```

Comment:
Are the two lean stories not a full copy of the normal stories? If yes, I think we should move into a common file, maybe a new one?

Moreover (maybe it comes too late ;) ). But I'm not convinced of using the home stories to demonstrate the lean mode. because for this design, the lean mode in real shouldn't be used and our representation looks somehow wrong. Maybe we create a custom story which imitates a business case (some forms or tables and header). I could even live with not even having any example for the lean mode as it isn't really something different. Probably to be discussed with @kyubisation